starlink/main.py

```diff
@@ -20,6 +21,7 @@
 
 schedule.every(1).hours.at(":00").do(run, icmp_ping).tag("Latency")
 schedule.every(1).hours.at(":00").do(run, grpc_get_status).tag("gRPC")
+schedule.every(1).hours.at(":00").do(run, grpc_get_location).tag("gRPC")
```

Comment:
some dishes might not have location access enabled. could you either add additional error handling or add an additional command line option / environment variable when running leoviz container to run `grpc_get_location`?